ios/PurchasesHybridCommon/ObjCAPITester/RCErrorContainerAPITests.m

```diff
@@ -0,0 +1,35 @@
+//
+//  RCErrorContainerAPITests.m
+//  ObjCAPITester
+//
+//  Created by Andrés Boedo on 4/13/22.
+//  Copyright © 2022 RevenueCat. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+@import PurchasesHybridCommon;
+@import PurchasesHybridCommonSwift;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface RCErrorContainerAPITests: NSObject
+
+@end
+
+
+@implementation RCErrorContainerAPITests: NSObject
+
+- (void)testAPI {
+
+    RCErrorContainer *container = [[RCErrorContainer alloc] initWithError:[[NSError alloc] init]
+                                                             extraPayload:@{}];
+    NSInteger code __unused = container.code;
```

Comment:
the `__unused` here prevent build warnings for creating a variable but not actually using it. in Swift we'd just do `_` for the variable name